pr_body.md

```
@@ -0,0 +1,10 @@
+﻿Implements present-LOD fallback (hysteresis + blend) with telemetry and CLI wiring.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)